test/test-tag.js

```diff
@@ -48,3 +48,14 @@ test('allow empty attributes', function (t) {
 
     t.end();
 });
+
+test('should expose getUserAttrs()', function (t) {
+    t.equal(typeof tag.getUserAttrs, 'function');
+    t.end();
+});
+
+test('tag.getUserAttrs() should get and return an object', function (t) {
+    t.throws(tag.getUserAttrs); // eslint-disable-line
```

Comment:
That's exactly the proper solution; the tests need to work in ES3, where `.throws` is a syntax error.